letsencrypt/build.yaml

```diff
@@ -17,3 +17,4 @@ args:
   CERTBOT_NETCUP_VERSION: 1.1.1
   CERTBOT_NJALLA_VERSION: 1.0.0
   CERTBOT_VERSION: 1.21.0
+  CERTBOT_DNS_IONOS_VERSION: 2022.05.15
```

Comment:
```suggestion
  CERTBOT_DNS_IONOS_VERSION: 2022.5.15
```
It looks like pip figures it out if you use `2022.05.15` but the version is actually `2022.5.15` according to pip:
![Screen Shot 2022-08-29 at 11 34 14 AM](https://user-images.githubusercontent.com/2037026/187238680-2a71d2f2-5457-438c-85f4-01cd4889d086.png)